### PR TITLE
pass back the dynamoDB json to the transformer function

### PR DESCRIPTION
**Pull request by dawez**
We had an issue in production where we need to access the raw dynamoDB JSON stream in the transform function.

This PR pass to the transform function the raw dynamoDB JSON argument.

### CODE REVIEW

**Comment by matrus2** [comment]
Hi, 

Thanks for using this library and submitting PR. Sorry for late reply.

File: src/index.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ exports.pushStream = async (
         const oldBody = record.dynamodb.OldImage ? converter(record.dynamodb.OldImage) : undefined
         body = removeEventData(body)
         if (transformFunction) {
-          body = await Promise.resolve(transformFunction(body, oldBody))
+          body = await Promise.resolve(transformFunction(body, oldBody, record.dynamodb.NewImage))
```
Comment (matrus2) [inline src/index.js]: Why not then pass entire record?

Comment (matrus2) [reply src/index.js]: Pls update readme file

Comment (matrus2) [reply src/index.js]: Would be nice to caver it by test

**Comment by dawez** [comment]
Sorry for the delay, I will submit the changes in the PR in next days.

**Comment by dawez** [comment]
Implemented requested changes and updated the readme. 

It seems that the Travis CI is failing alhought, can you please check why? Thanks.

File: src/index.js (hunk 1 above)
Comment (dawez) [reply src/index.js]: done

**Comment by dawez** [comment]
@matrus2 is there anything else that I should change in the PR?

**Comment by matrus2** [comment]
Sorry, somehow missed this one. I will look asap.

File: README.md [hunk 2]
```diff
@@ -61,17 +61,38 @@ Upload Lambda to AWS and _star_ this repository if it works as expected!!
...
 ```bash
-docker run -i -p 9200:9200 --name my_elastic -p 9300:9300 -e "discovery.type=single-node" elasticsearch
+docker run -i -p 9200:9200 --name my_elastic -p 9300:9300 -e "discovery.type=single-node" elasticsearch:7.2.0
+```   
+### Running tests
+
+Tests are written in using mocha [https://mochajs.org/]. In order to run them ensure that mocha command is available on the path:
+
+```bash
+npm install -g mocha
```
Comment (matrus2) [inline README.md]: You don't need to install it globally it is in package.json

File: README.md [hunk 3]
```diff
@@ -61,17 +61,38 @@ Upload Lambda to AWS and _star_ this repository if it works as expected!!
...
+```                 
+
+Head in the test folder and launch
+
+```bash
+mocha index.js
```
Comment (matrus2) [inline README.md]: npm test will do the trick

File: README.md (hunk 2 above)
Comment (dawez) [reply README.md]: Ah right, will change the readme then.

**Comment by dawez** [comment]
Thank you for the review, I changed the readme.md.

**Pull request merged**